tests/state_test_util.go

```diff
@@ -151,33 +151,33 @@ func (t *StateTest) Subtests() []StateSubtest {
 }
 
 // Run executes a specific subtest and verifies the post-state and logs
-func (t *StateTest) Run(ctx context.Context, tx ethdb.Database, subtest StateSubtest, vmconfig vm.Config) (*state.IntraBlockState, error) {
-	state, root, err := t.RunNoVerify(ctx, tx, subtest, vmconfig)
+func (t *StateTest) Run(ctx context.Context, tx ethdb.Database, subtest StateSubtest, vmconfig vm.Config) (*state.IntraBlockState, *state.TrieDbState, error) {
+	state, root, tds, err := t.RunNoVerify(ctx, tx, subtest, vmconfig)
```

Comment:
Please do not bring back `tds`, but compute state root using `turbo/trie` package. We would like to remove `tds` as soon as possible